ecommerce/extensions/basket/views.py

```diff
@@ -184,7 +184,7 @@ def _get_course_data(self, product):
             )
 
         return {
-            'product_title': course_name,
+            'product_title': product.title,
```

Comment:
course_name = course.get('title', '') This returns default value '', the only way that course_name returns None is the try-catch block is failing, no? @Alec4r

Reply:
@Squirrel18 fails because there is no course runs

Reply:
As far as I know, we don't want course_name to be a blank value (it comes from discovery service), reason why it was changed to display the product name which is set when the course product is create (normally this name is related to the course, or is the same course name)

Reply:
@Squirrel18

Reply:
ok, i didn't know where course_name came from. @jfavellar90 @Alec4r